src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -635,25 +635,16 @@ private BoundNode IntroduceFrame(BoundNode node, LambdaFrame frame, Func<ArrayBu
 
             var prologue = ArrayBuilder<BoundExpression>.GetInstance();
 
-            BoundExpression newFrame;
-            if (frame.Constructor == null)
-            {
-                Debug.Assert(frame.TypeKind == TypeKind.Struct);
-                newFrame = new BoundDefaultExpression(syntax: syntax, type: frameType);
```

Comment:
I always forget what the difference here is between the frame and the frame pointer. What does it mean to issue a default expression for the frame? I can see what it would mean to initialize the synthesized local, but I don't know what it means for the frame itself.